google/cloud/firestore_v1/base_aggregation.py

```diff
@@ -246,21 +258,30 @@ def get(
                 should be retried.  Defaults to a system-specified policy.
             timeout (float): The timeout for this request.  Defaults to a
                 system-specified value.
+            explain_options
+                (Optional[:class:`~google.cloud.firestore_v1.query_profile.ExplainOptions`]):
+                Options to enable query profiling for this query. When set,
+                explain_metrics will be available on the returned generator.
 
         Returns:
-            list: The aggregation query results
+            QueryResultsList[AggregationResult] | Coroutine[Any, Any, List[AggregationResult]]:
+                The aggregation query results.
 
         """
 
     @abc.abstractmethod
     def stream(
         self,
         transaction: Optional[transaction.Transaction] = None,
-        retry: Optional[retries.Retry] = gapic_v1.method.DEFAULT,
+        retry: Union[
+            retries.Retry, None, gapic_v1.method._MethodDefault
+        ] = gapic_v1.method.DEFAULT,
         timeout: Optional[float] = None,
+        *,
+        explain_options: Optional[ExplainOptions] = None,
     ) -> (
-        Generator[List[AggregationResult], Any, None]
-        | AsyncGenerator[List[AggregationResult], None]
+        StreamGenerator[Optional[List[AggregationResult]]]
+        | AsyncStreamGenerator[List[AggregationResult], Any, None]
```

Comment:
does AsyncStreamGenerator exist in this PR?

Reply:
I think `stream()` does return AsyncStreamGenerator, [see here](https://github.com/googleapis/python-firestore/blob/ea64f23a68d7610ce7ed3369c723e2cbffd2c56d/google/cloud/firestore_v1/async_aggregation.py#L164).

Reply:
But I removed the async return value :)